src/CanService.h

```diff
@@ -24,7 +24,7 @@ class CanService : public Service
 
   virtual void process(UserInterface::RequestedAction requestedAction) override;
   virtual Processed handleRawMessage(CANFrame *msg) override;
-  Processed handleMessage(unsigned int opc, CANFrame *msg) override;
+  virtual Processed handleMessage(unsigned int opc, CANFrame *msg) override;
```

Comment:
Marking all overridden methods with "virtual" so it is easier to find the virtual methods in a class.